src/blockdata/witness.rs

```diff
@@ -0,0 +1,395 @@
+//! Witness
+//!
+//! This module contains the [`Witness`] struct and related methods to operate on it
+//!
+
+use consensus::encode::{Error, MAX_VEC_SIZE};
+use consensus::{Decodable, Encodable, WriteExt};
+use io::{self, Read, Write};
+use prelude::*;
+use VarInt;
+
+#[cfg(feature = "serde")]
+use serde;
```

Comment:
Nit: this seems redundant (we always have access to the crate without `use`ing it)